spec/ParseQuery.spec.js

```diff
@@ -3850,7 +4141,6 @@ describe('Parse.Query testing', () => {
       })
       .then(function (savedFoobar) {
         const foobarQuery = new Parse.Query('Foobar');
-        foobarQuery.include('barBaz');
```

Comment:
@dplewis I removed this because there are already tests for `include`, and we are forcing `include`